.travis.yml

```diff
@@ -1,4 +1,9 @@
-language: c
-compiler: gcc
```

Comment:
On 5 September 2015 at 02:33, Dmitry Shachnev wrote:
> > @@ -1,4 +1,9 @@
> > -language: c
> > -compiler: gcc
>
> Unfortunately Travis doesn't seem to support different languages. So you can either specify multiple compliers or multiple Python versions.
>
> I wanted to test with both Python 2.7 and Python ≥ 3.3, but now as you committed 30e10d9 to specify multiple compilers, I reverted my change and it tests only Python 2.7 now.
>
> If in future Travis moves to a newer OS that has Python ≥ 3.3 (Ubuntu 12.04 doesn't), we will be able to install it using apt-get and test with it.

You should be able to achieve that using multiple entries for the same
compiler with different `env:` settings in each.

Something like this (inspired by swig's `.travis.yml`):

```
compiler:
  - clang
  - gcc
    env: PYTHON=python2.7
  - gcc
    env: PYTHON=python3.2
  - gcc
    env: PYTHON=python3.3
```

Cheers,
    Olly